pkgs/development/libraries/libvmaf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, autoconf, automake, intltool, libtool, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "libvmaf";
+  version = "1.3.15";
+
+  src = fetchurl {
+    url="https://github.com/Netflix/vmaf/archive/v1.3.15.tar.gz";
+    sha256="13ajbcidngjvgl0rr7l0mb43h651p5pqj3d1linrfk9c222b9fs3";
+  };
+
+  buildInputs = [ autoconf automake intltool libtool pkgconfig ];
```

Comment:
if they are just needed for the build, then they should be nativeBuildInputs. buildInputs is usually for package that you need to link against for runtime.
```suggestion
  nativeBuildInputs = [ autoconf automake intltool libtool pkgconfig ];
```

Reply:
Also, you could probably replace all of that with `[ autoreconfHook pkgconfig ];`